indico/modules/rb_new/controllers/backend/rooms.py

```diff
@@ -106,3 +106,17 @@ def _process_PUT(self):
     def _process_DELETE(self):
         session.user.favorite_rooms.discard(self.room)
         return '', 204
+
+
+# TODO
+class RHRoomUpdate(RHRoomBase):
+    # TODO: Check if current user is allowed to edit room
+    def _process_args(self):
+        self.room = Room.get_one(request.view_args['room_id'])
```

Comment:
`RHRoomBase` does that already. also, let's add a check for rb admin privs for now so we don't accidentally forget it later

Reply:
After updating the commit I see that instead of using RHRoomBase (as I intended) I used RHRoomBookingBase. I'll check this is out tomorrow morning!